Backend/app/auth/auth_service.py

```diff
@@ -163,68 +155,56 @@ def get_current_user(
 
 
 def hash_password(plain_password: str) -> bytes:
-    """Hash a password with a randomly-generated salt
+    """Hashes a password using bcrypt with a random salt.
 
     Args:
-    ----
-        plain_password (str): plain text password
+       plain_password: Password in plaintext to hash.
 
     Returns:
-    -------
-        bytes: the hashed password
-
+       Bytes containing the hashed password.
     """
     return bcrypt.hashpw(plain_password.encode(), bcrypt.gensalt())
 
 
 def verify_password(plain_password: str, hashed_password: bytes) -> None:
-    """Verifies if plan text password is the same as a hashed password
+    """Verifies if a plaintext password matches its hashed version.
 
     Args:
-    ----
-        plain_password (str): plain text password
-        hashed_password (bytes): hashed password
+       plain_password: Password in plaintext.
+       hashed_password: Encrypted password bytes to compare against.
 
     Raises:
-    ------
-        VerifyPasswordException: if passwords don't match
-
+       VerifyPasswordException: If the passwords do not match.
     """
     if not bcrypt.checkpw(plain_password.encode(), hashed_password):
         raise VerifyPasswordException
 
 
 def get_token_expire_date() -> datetime:
-    """Returns expire date for new token
-
-    Returns
-    -------
-        datetime: the expire date for the token
+    """Calculates the expiration date for a new token.
 
+    Returns:
+       Datetime object representing when the token will expire.
     """
-    current_utc_datetime = datetime.now(timezone.utc).replace(tzinfo=timezone.utc)  # noqa: UP017 TODO
```

Comment:
This won't work, that's why `# noqa: UP017 TODO` flag is set. We have a bug related to this change in the past

Reply:
Got it, reverting...